doc/user/content/get-started/install-materialize-emulator.md

```diff
@@ -59,21 +59,24 @@ not suitable for production deployments</redb>.
    been already downloaded.
 
    ```sh
-   docker run -d -p 127.0.0.1:6875:6875 -p 127.0.0.1:6876:6876 materialize/materialized:{{< version >}}
+   docker run -d -p 127.0.0.1:6874:6874 -p 127.0.0.1:6875:6875 -p 127.0.0.1:6876:6876 materialize/materialized:{{< version >}}
    ```
 
    When running locally:
 
    - The Docker container binds exclusively to localhost, for security reasons.
+   - The web console is available on port `6874`.
```

Comment:
```suggestion
   - The [Materialize console](/console/) is available on port `6874`.
```

Reply:
@kay-kim, we seem to use both "Materialize Console" and "Materialize console" in the documentation. Can we decide on one variation and stick to it?

Reply:
Yup -- going to go with Materialize Console (capitalizing both).